src/python/pants/backend/python/goals/coverage_py.py

```diff
@@ -186,8 +186,8 @@ def _validate_and_update_config(
         )
     coverage_config.set("run", "relative_files", "True")
     omit_elements = [em for em in run_section.get("omit", "").split("\n")] or ["\n"]
-    if "test_runner.pex/*" not in omit_elements:
-        omit_elements.append("test_runner.pex/*")
+    if "pytest.pex/*" not in omit_elements:
+        omit_elements.append("pytest.pex/*")
```

Comment:
Should this rule be getting this filename from a constant somewhere?

Reply:
Maybe, we would factor up this line:

https://github.com/pantsbuild/pants/blob/b9c7c4d91bab7032291c493eae3e7573813f96f6/src/python/pants/backend/python/goals/pytest_runner.py#L120-L123

Wdyt? Probably worth it to make explicit the dependency.

Reply:
Yea... I stopped short of suggesting a subsystem dependency... just a constant should be fine?

Reply:
Oh, this results in a circular dependency. Going to stick with this, since the tests will catch this error in the future.